### ISSUE TITLE
`gozen create`  command creates a blank project without warnings

### ISSUE THREAD

**Issue by kernelshard** [tech-thinker/gozen#8]
I encountered an issue when using the gozen create command. The command executed successfully and created a project, but it did not provide any warnings about the missing required fields. Specifically, both app_name and package_name were left blank in the generated gozen.json file. The package_name should be a required field, and the command should warn the user if it is not provided.

_Steps to Reproduce:_

**Run the command:**
```gozen create```

Observe the output:

```Project created at location: ./```

### Check the contents of gozen.json:
```json
{
    "app_name": "",
    "package_name": "",
    "driver": "sqlite"
}
```

### Expected Behavior:

The command should not allow the creation of a project with empty app_name and package_name fields.
The command should provide a warning or error message indicating that package_name is required.
### Actual Behavior:

The command creates a project with empty fields in `gozen.json` without any warnings.
### Additional Information:

Please ensure that `package_name` is enforced as a required field in the command implementation.
A user-friendly message should guide users to provide the necessary information before proceeding with project creation.
Thank you for your attention to this matter!

**Comment by mrasif** [comment]
Thank you for your valuable feedback, @samims ! We will investigate this issue and work on releasing a patch shortly.

**Comment by mrasif** [comment]
@samims fixes released on version `v0.1.2`.
